http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -95,6 +94,7 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
   override def createAndPollRuntime(msg: CreateAzureRuntimeMessage)(implicit ev: Ask[F, AppContext]): F[Unit] =
     for {
       ctx <- ev.ask
+      _ <- logger.info(s"[AzurePubsubHandler/createAndPollRuntime] beginning for runtime ${msg.runtimeId}")
```

Comment:
Do you really want / need these additional logs or were they for debugging purposes only?

Reply:
I think the more logging the better personally, especially since these methods are massive and make multiple external api calls